test/OpenTelemetry.Tests/Shared/FileHelper.cs

```diff
@@ -0,0 +1,71 @@
+// <copyright file="FileHelper.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Tests.Shared;
+
+/// <summary>
+/// This class contains methods to help with common file I/O tasks.
+/// </summary>
+internal static class FileHelper
```

Comment:
I still don't understand why we would need to introduce this level of complexity to simple test cases, can this be simplified?